orbstation/code/objects/vending/vendors.dm

```diff
@@ -29,6 +29,16 @@
     orb_premium = list(
         /obj/item/toy/plush/pennyplushie = 1,
     )
+
+/obj/machinery/vending/wardrobe/robo_wardrobe
+	orb_premium = list(
+		/obj/item/toy/plush/fishplushie = 1
+	)
+
+/obj/machinery/vending/wardrobe/cargo_wardrobe
+	orb_premium = list(
+		/obj/item/toy/plush/ivonaplushie = 1
```

Comment:
This will not actually work, since the cargo wardrobe's orb_premium is already defined in this same file. Add it to that list instead.

Reply:
i can make this change but it was totally working fine when i tested it